extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -16,10 +17,12 @@ require([
 	'ext.wikia.adEngine.provider.yavliTag',
 	'wikia.window',
 	'wikia.loader',
-	require.optional('ext.wikia.adEngine.recovery.gcs')
+	require.optional('ext.wikia.adEngine.recovery.gcs'),
+	require.optional('ext.wikia.adEngine.template.floatingRail')
 ], function (
 	adContext,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 18 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AExcessiveParameterList)